helm-files.el

```diff
@@ -3470,28 +3470,28 @@ Colorize only symlinks, directories and files."
                                          (string-match ffap-url-regexp i)))
                                (not (string-match helm-ff-url-regexp i)))
                           (helm-basename i) i)
-           for type = (car (file-attributes i))
```

Comment:
Perhaps just check if file (i) is remote and `helm-ff-tramp-not-fancy` e.g
for type = (and helm-ff-tramp-not-fancy (not (file-remote-p i)) (car (file-attributes i)...